tests/e2e/e2e_common.go

```diff
@@ -120,6 +120,7 @@ const (
 	envVmdkDiskURL                             = "DISK_URL_PATH"
 	vsanDefaultStorageClassInSVC               = "vsan-default-storage-policy"
 	vsanDefaultStoragePolicyName               = "vSAN Default Storage Policy"
+	busyBoxImage                               = "gcr.io/google_containers/busybox:1.24"
```

Comment:
just one comment, otherwise LGTM
change busyBoxImage -> busyBoxImageOnGcr or something like that
and CreatePod to createPod just like all the other methods in the util.

Reply:
One more comment, can you do the same for statefulsets and deployments as well.

Reply:
done